src/pydap/parsers/dmr.py

```diff
@@ -169,35 +177,80 @@ def dmr_to_dataset(dmr):
         for dim in variable["dims"]:
             variable["shape"] += (variables[dim]["size"],)
 
-    # Convert the ordered dictionary to dataset
-    dataset_name = dom_et.attrib["name"]
-    dataset = pydap.model.DatasetType(dataset_name)
     for name, variable in variables.items():
-        data = DummyData(dtype=variable["dtype"], shape=variable["shape"])
+        var_name = variable["name"]
+        path = None
+        if len(var_name.split("/")) > 1:
+            # path-like name - Groups!
+            parts = var_name.split("/")
+            var_name = parts[-1]
+            path = ("/").join(parts[:-1])
+            # need to do the same with dimensions
+            for i in range(len(variable["dims"])):
+                dim = variable["dims"][i].split("/")[-1]
+                variable["dims"][i] = dim
+            variable["attributes"]["path"] = path
+
+        data = DummyData(dtype=variable["dtype"], shape=variable["shape"], path=path)
         array = pydap.model.BaseType(
-            name=variable["name"], data=data, dimensions=variable["dims"]
+            name=var_name,
+            data=data,
+            dimensions=variable["dims"],
         )
         if variable["has_map"]:
-            var = pydap.model.GridType(name=variable["name"])
-            var[name] = array
+            var = pydap.model.GridType(name=var_name)
+            var[var_name] = array
             for dim in variable["dims"]:
                 var[dim] = copy.copy(dataset[dim])
         else:
             var = array
         var.attributes = variable["attributes"]
-        dataset[var.name] = var
+        if "parent" in variable.keys() and variable["parent"] in [

```

Comment:
This works in the case variable name is oath like, but the variable belongs to a sequence or structure. For example:
```python
Group1/Group2/Seq_name/Var_name
```
Then the variable name is defined as `Seq_name.Var_name`, and path='Group1/Group2'. Similarly for a structure